targets/ESP32/_nanoCLR/nanoFramework.Hardware.Esp32.Rmt/nanoFramework_hardware_esp32_rmt_native_nanoFramework_Hardware_Esp32_Rmt_RmtChannel.cpp

```
@@ -1,144 +1,214 @@
-#else
-    stack.SetResult_I4(80000000);
+    registeredRmtChannels.erase(handle);
+
+#ifdef DUMP_RMT_CHANNELS
+    DumpRegisteredRmtChannels("Erase channel");
 #endif
+}
 
-    NANOCLR_NOCLEANUP_NOLABEL();
+HRESULT Library_nanoFramework_hardware_esp32_rmt_native_nanoFramework_Hardware_Esp32_Rmt_RmtChannel::
+    RmtMapEspErrToClrErr(esp_err_t err)
```

Cppcheck (2.21.0)

[style] The function 'RmtMapEspErrToClrErr' is never used.

(unusedFunction)